AutomatedTesting/Gem/PythonTests/Atom/atom_utils/atom_constants.py

```diff
@@ -271,11 +286,17 @@ def camera(property: str = 'name') -> str:
     def cube_map_capture(property: str = 'name') -> str:
         """
         CubeMap capture component properties.
+          - 'Specular ILB' controls the quality of Specular IBL created
+          - 'Capture Type': controls if CubeMap Capture component uses 'Diffuse ILB' or 'Specular ILB'
+          - 'Exposure': Controls the exposure light in the image taken
         :param property: From the last element of the property tree path. Default 'name' for component name string.
         :return: Full property path OR component name if no property specified.
         """
         properties = {
-            'name': 'CubeMap Capture'
+            'name': 'CubeMap Capture',
+            'Specular IBL': 'Controller|Configuration|Specular IBL CubeMap Quality',
```

Comment:
short name (key) should be either the whole final path element or descriptive of this being quality level
by convention this would be 'Specular IBL CubeMap Quality'

Reply:
updated